### DISCUSSION TITLE
Multiple alarm panels

### DISCUSSION THREAD

**Discussion by plakkiesza** [ParadoxAlarmInterface/pai#354] [Q&A] [answered]
I have to monitor three premises. Ive been looking at how I can connect all three alarm systems to home assistant. It does not look like pai can connect to more than 1 panel at a time. I was then thinking that I would just run 3 pai instances. Can I modify the mqtt topics for two of the pai instances?

**Comment by yozik04** [comment, marked as answer]
You can run any number of PAI instances you want.
You need to have a unique MQTT_BASE_TOPIC for each.

**Reply by plakkiesza** [reply]
Thank you, not sure how I missed that.

**Reply by ReubenTechery** [reply]
Hi I am trying to add 4 alarm panels. I have added one and it all works well but really am unsure how to add the other 3 . I dont know how to add more then 1 Pai instance nor do i know how to add a unique Mqtt_base_topic for each. Are you able to elaborate? that would be super helpful! Thanks

**Reply by yozik04** [reply]
You have panel overflow. I've added easy addition of two in HomeAssistant OS by having two plugins installed. One plugin per panel. You can have as many panels as you want, but you need to control docker yourself then, out of HomeAssistant.

**Comment by ReubenTechery** [comment]
> You have panel overflow. I've added easy addition of two in HomeAssistant OS by having two plugins installed. One plugin per panel. You can have as many panels as you want, but you need to control docker yourself then, out of HomeAssistant.

Sorry I am an absolute beginner at all of this. All i have is mqtt broker + PAI addons installed , I have 4 alarms all on the same LAN as the home assistant. I can only input details for 1 alarm through PAI and mqtt broker. I am struggling to find out how to I guess duplicated the PAI addon 3 more times and set it up with mqtt. Im not sure if this even makes sense sorry but am really trying to get it going
